sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/SetLocaleForDataTypes.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Data;
+using System.Globalization;
+
+namespace Tests.Diagnostics
```

Comment:
Add more tests:
1. Create DataTable and sets the Locale property on some different object.
2. Create two DataTable/Sets, but set the Locale property only on one of them.
3. DataTable dt; dt = new DataTable(); dt.Locale=...

Should we to check if the culture is set to invariant (I presume not)?